test/data/datasets.js

```diff
@@ -3,6 +3,12 @@ import { comparator } from 'ramda';
 import { dataStore } from './data-store';
 import { extendedProjects } from './projects';
 
+const normalizeProperty = (property) => ({
+  odataName: property.name,
```

Comment:
`odataName: property.odataName ?? property.name`

Reply:
`property.name` is the default `odataName`. The defaults are merged with `property` using `...property`, so if `property` specifies its own `odataName`, it will overwrite the default.